src/ducks/settings/AddAccountLink.jsx

```diff
@@ -26,11 +24,7 @@ class SameWindowLink extends Component {
       }
     )
 
-    if (__TARGET__ === 'browser') {
-      window.location = redirectionURL
-    } else if (__TARGET__ === 'mobile') {
-      open(redirectionURL)
-    }
+    window.location = redirectionURL
```

Comment:
I think it would be worth commenting why we do not use open here. Apparently it is because in the onboarding flow, the user clicks on email links, which open the "native" browser (external Firefox for example) instead of the in-app one. This means login cookies are stored in the external browser. To prevent asking the user to login again, we have to use an external browser instead of the in app browser.